ftplugin/mail_vifm.vim

```diff
@@ -52,7 +52,7 @@ function! s:HandleRunResults(exitcode, listf)
 
 	if filereadable(a:listf) && l:insert_pos != 0
 		for line in readfile(a:listf)
-			call append(l:insert_pos, 'Attach: '.line)
+			call append(l:insert_pos, 'Attach: '.substitute(line, " ", "\\\\ ", "g"))
```

Comment:
Can you change this to use `escape()` function? I think it's more appropriate here.

Reply:
I've made the update as requested. Was not familiar with that function. Thanks!